src/components/DiceModule/Dice.tsx

```diff
@@ -16,7 +17,7 @@ function Dice({
 }: DicePropTypes
 ) {
 	const DropdownContent = () => {
-		const maxDiceInOneRoll = diceType === D6_CONAN ? 15 : 10;
+		const maxDiceInOneRoll = diceType === (D6_CONAN || D6_INFINITY) ? 15 : 10;
```

Comment:
This will never work for `D6_INFINITY`. 
```suggestion
		const maxDiceInOneRoll = (diceType === D6_CONAN || diceType === D6_INFINITY) ? 15 : 10;
```

Reply:
This appears to set the max die for Infinity to 10.  I am not sure if I even need to add the OR check here for Infinity... Should I simply removed this addition for Infinity then?